views/search/search.html

```diff
@@ -9,4 +9,5 @@ <h4 class="jw-content-title">
 
   <jw-card-grid feed="vm.feed" on-card-click="vm.cardClickHandler" card-class-name="jw-lazy-load"
       cols="{xs:2, sm:2, md:3, lg:4, xl:5}"></jw-card-grid>
-</div>
\ No newline at end of file
+  <jw-button ng-if="vm.itemsLeft()" ng-click="vm.showMoreClickHandler()" class="jw-button-default jw-button-secondary jw-button-show-more jw-card-grid">Show more results...</jw-button>
```

Comment:
Why is there a `jw-card-grid` class name on this button?

Reply:
Because it needs the same margins as the grid above. It will also give the right margins on mobile. Better to reuse a css class then create a new one that does the same

Reply:
@ChristiaanScheermeijer Suggesting to add an alias for this class name. So it makes mores sense semantically.

Reply:
@brniamut Semantically, it's better to create a new class. Even if it means re-using code.

Please move this to a new selector.

Reply:
Yes, agree